### DISCUSSION TITLE
Error installing from helm on self hosted kubernetes cluster

### DISCUSSION THREAD

**Discussion by aroekene** [Budibase/budibase#4398] [Help]
I've been trying to install budibase on our self hosted kubernetes cluster, but couchdb goes into CrashLoopBackOff.
Kubernetes v1.21.5 and containerd 1.5.5
Installed with default values.yaml and helm command helm install --create-namespace --namespace budibase budibase budibase/budibase

Attached the log file for couchdb
[couch.log](https://github.com/Budibase/budibase/files/8028251/couch.log)

**Comment by johnnycheetah** [comment]
did you find a fixfor this or figure out the issue? am having a similar issue.
